### ISSUE TITLE
Break "Tools" up into multiple categories

### ISSUE THREAD

**Issue by lilyinstarlight** [CUCyber/hackpack#65]
The "Tools" catagory is too broad and should be broken up into categories, for example "Sniffing", "Services and Applications", "Operating System Concepts".

**Comment by robertu94** [comment]
@fkmclane 
The "Tools" category is broken up into multiple sections.  Currently it is split into shells, provisioners, exploration, and openssl.  Perhaps the exploration section should be renamed, and openssl should be moved or renamed, but I don't quite see the purpose of this issue.

Also "Services and Applications"  probably fits in section entitled "Services"

What did you have in mind for "Operating System Concepts"?

**Comment by lilyinstarlight** [comment]
The shell would fit in "Operating System Concepts", but I'm very open to different names for sections.

The tools section is no longer split up. See #66. I did not see what was entirely different between parts in the tools section and parts in other sections, so I did not add functionality for more than one level of subsections. There is nothing preventing that from happening as the whole section thing depends on ordering of the markdown files, but I don't find subsubsections useful.
